src/azure-cli/service_name.json

```diff
@@ -0,0 +1,688 @@
+[
+  {
+    "Command": "az account",
+    "Description": "Manage Azure subscription information.",
+    "AzureServiceName": "Azure core",
+    "ConversationalServiceName": "Azure core",
+    "URL": "https://docs.microsoft.com/azure/cost-management-billing/manage/create-subscription"
+  },
+  {
+    "Command": "az acr",
```

Comment:
this file will be manually maintained in future right?

Reply:
Yes